working-groups.md

```diff
@@ -0,0 +1,63 @@
+# Working Groups 
+
+### [Business Planning][biz-wg]
+
+WG for stewarding our business strategy, this includes generating and pursuing business opportunities, relationships, branding, and communications.
+
+📅 Calls: [Weekly][calendar]  
+👪 Members: [**@business-planning-wg**][t-biz]  
+✅ Tasks: GitHub board > [`wg:business-planning` label][l-biz]  
+
+### [Finance][fin-wg]
+
+WG for stewarding the financial sustainability of the organization, this includes establishing accounting practices, maintaining financial accounts, processing invoices, expenses, and payroll, and creating reports on our financial statements and tax obligations.
+
+📅 Calls: [Weekly][calendar]   
+👪 Members: [**@finance-wg**][t-fin]  
+✅ Tasks: GitHub board > [`wg:finance` label][l-fin]  
+
+### [Governance][gov-wg]
+
+WG for drafting by-laws, articles of incorporation, and planning for the first General Meeting (and subsequent structure).
```

Comment:
Slightly more current take: 
```suggestion
WG for ensuring healthy structure of our cooperative, this includes drafting by-laws, maintaining articles of incorporation, and planning annual members meetings.
```

Reply:
Okay, per @benhylau, I'll merge in governance, merge this PR, and expect infra to update shortly